.changes/v3.6.0/818-notes.md

```diff
@@ -1,2 +1,2 @@
 * Improve docs for `vcd_nsxt_network_dhcp` VDC Group support [GH-818]
-* Improves VDC Group guide documentation [GH-818]
\ No newline at end of file
+* Improves VDC Group guide documentation [GH-818]
```

Comment:
Why all the changes above? The viewer doesn't seem to show a diff :)

Reply:
Missing newlines, broke whole CHANGELOG